redaxo/src/core/lib/sql/util.php

```diff
@@ -84,6 +84,10 @@ public static function organizePriorities($tableName, $prioColumnName, $whereCon
      */
     public static function importDump($file, $debug = false)
     {
+        if ('.sql' != substr($file, -4, 4)) {
```

Comment:
Zur Info, wir haben bereits das Polyfill-Paket für PHP 8 drin, daher könnten wir auch bereits die neuen Funktionen nutzen:

```suggestion
        if (str_ends_with($file, '.sql')) {
```